app/jobs/create_tap_activity.rb

```diff
@@ -0,0 +1,20 @@
+require 'notifications/client'
+
+class CreateTapActivity < ApplicationJob
+  class UnableToCreateSummaryDocumentActivity < StandardError; end
+
+  queue_as :default
+
+  def perform(appointment_summary, guider)
+    telephone_appointment = TelephoneAppointments::SummaryDocumentActivity.new(
+      appointment_id: appointment_summary.reference_number,
+      owner_uid: guider.uid,
+      delivery_method: appointment_summary.requested_digital ? 'digital' : 'postal'
+    )
+
+    return if telephone_appointment.save
+
+    errors = telephone_appointment.errors.map { |field, error| "#{field}: #{error}" }.join(', ')
+    raise UnableToCreateSummaryDocumentActivity, "Errors: #{errors}"
```

Comment:
Also, this will just keep retrying - which doesn't make sense if we get validation errors back - as it won't be recoverable without intervention. The right thing to do here is report the error to bugsnag so we can follow up there.